app/packages/prometheus/src/components/PrometheusPage.tsx

```diff
@@ -0,0 +1,432 @@
+import {
```

Comment:
found a bug, which traps the user in the UI, steps to reproduce:

1. add a valid query `chi_clickhouse_metric_NumberOfDetachedParts` and execute
2. click the `+` icon
3. click on `search` or hit `shift + enter`
4. click the `-` icon
5. try to execute the query

![image](https://user-images.githubusercontent.com/60616628/224669782-fa5eb2c5-8081-4d40-bf47-672fc1d0d6aa.png)